hoot-core/src/main/cpp/hoot/core/algorithms/changeset/ChangesetDeriver.h

```diff
@@ -65,7 +65,7 @@ class ChangesetDeriver : public ChangesetProvider
   /**
    * @see ChangeSetProvider
    */
-  virtual Change readNextChange();
+  virtual Change readNextChange() override;
```

Comment:
Again `close` and `hasMoreChanges` also are virtual function from the base class but may not be triggering SonarCube findings.  Seems odd.